bin/problem.py

```diff
@@ -471,7 +473,7 @@ def validate_format(problem, validator_type, constraints=None):
             log('Not validating .ans for interactive problem.')
             return True
 
-        if not validators:
+        if validators is False:
```

Comment:
Hrmm; I'm a bit conflicted on this: in this case, the `problem.validators()` function will already give an error, so it could also just return `[]` instead of `False`. That way, we wouldn't have to catch the `false` here and could just loop over the empty list. The `validate_format` function would always return true in that case, but that's not really an issue anyway I think, since calling `error` at least once makes the program return non-zero already.

But anyway, that's for another PR.